tests/cases/dom/events/off.js

```diff
@@ -61,3 +84,42 @@ QUnit.test(
         );
     }
 );
+
+QUnit.test(
+    "off() with an array of elements and an invalid event",
+    (assert) =>
+    {
+        assert.throws(
+            () => {
+                off([findOne(".example")], null, () => {});
```

Comment:
That is a type error and we shouldn't check that explicitly.

Reply:
That is absolutely correct. Updated and replaced with `""`.